topics/python/solutions/sort_solution.md

```diff
@@ -0,0 +1,26 @@
+## Algorithms
```

Comment:
Algorithms is quite generic header, maybe something with "sort"?

Reply:
Ok